sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Monitor.OpenTelemetry.Exporter.Models;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using Xunit;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter
+{
+    public class MetricHelperTests
+    {
+        [Fact]
+        public void ValidateMetricTelemetryItem()
+        {
+            var metrics = new List<Metric>();
+
+            using var meter = new System.Diagnostics.Metrics.Meter(nameof(ValidateMetricTelemetryItem));
+            using var provider = Sdk.CreateMeterProviderBuilder()
+                .AddMeter(meter.Name)
```

Comment:
where is this `meter` defined?

Reply:
It is defined in line 21.

Reply:
😄 sorry i missed !

Reply:
Good reminder to schedule annual eye check appointment :)